Loop/Managers/Alerts/StoredAlert.swift

```diff
@@ -16,13 +17,15 @@ extension StoredAlert {
           
     convenience init(from alert: Alert, context: NSManagedObjectContext, issuedDate: Date = Date()) {
         do {
-            self.init(context: context)
+            let name = String(describing: type(of: self))
```

Comment:
What is this code for?

Reply:
While running the unit tests, I kept seeing the following warnings:
```
CoreData: warning: Multiple NSEntityDescriptions claim the NSManagedObject subclass 'Loop.StoredAlert' so +entity is unable to disambiguate.
```
Note that these happened even before all of my changes.  This SO gives an example of how to mitigate this, which is what this is: https://stackoverflow.com/a/54126839